aten/src/ATen/native/LinearAlgebra.cpp

```diff
@@ -141,7 +141,9 @@ static void check_1d(const Tensor& t, const char* arg, const char* fn) {
 Tensor addr(const Tensor& self, const Tensor& vec1, const Tensor& vec2, Scalar beta, Scalar alpha) {
   check_1d(vec1, "vec1", "addr");
   check_1d(vec2, "vec2", "addr");
-  return at::_addr(self, vec1, vec2, beta, alpha);
+  Tensor b_self;
+  std::tie(b_self) = expand_size(self, {vec1.size(0), vec2.size(0)}, "addr");
```

Comment:
I see that `expand_size(self, sizes)` returns a 1-tuple but I'm not sure whyy

Reply:
oh it's just for codegen simplicity, so the codegen can just always be written in terms of tuples.  We could change that later, not sure if anything else is calling into that function.